src/website/shared/models/cve.py

```diff
@@ -287,6 +289,7 @@ class IssueStatus(models.TextChoices):
     WONTFIX = "W", _("wontfix")
 
 
+@pghistory.track(model_name="NixpkgsIssueSnapshot")
```

Comment:
We don't want to snapshot everything, as that will require a lot of churn and diffing the snapshots and such. Also the DB can be expected to be large. What we need to track already:
- status changed
- description snapshots
- cve added/removed
- derivation added/removed

All this indexed by user.

Reply:
I added tracking per field. pghistory already diffs changes, instead of snapshotting the whole row. 

For the CVE and derivation addition/removal I'll need to have clear the representation to contruct the proper query, for now it tracks the field directly in the NixpkgsIssue model.